games/o.yaml

```
@@ -1526,6 +1541,21 @@
   updated: 2019-11-20
   url: http://openspades.yvt.jp/
 
+- name: OpenSolomonsKey
+  originals:
+  - Solomon's Key
+  type: clone
+  repo: 'https://github.com/mdodis/OpenSolomonsKey'
+  url: 'https://github.com/mdodis/OpenSolomonsKey'
+  development: very active
+  status: playable
+  content: open
+  lang:
+  - C++
+  license:
+  - PD
```

```suggestion
  - Unlicense
```
and add it to the schema in `games.yaml`